Update 8.13.0.asciidoc (#106803)
elastic/elasticsearch: docs/reference/release-notes/8.13.0.asciidoc

[[breaking-8.13.0]]
[float]
=== Breaking changes

> There are no breaking changes in 8.13

Reviewer: ```suggestion
There are no breaking changes in 8.13.
```

> === Notable changes

Reviewer: ```suggestion
[[notable-changes-8.13.0]]
[float]
=== Notable changes
```

> There are notable changes in 8.13 that you need to be aware of but that we do not consider breaking, items that we may consider as notable changes are

Reviewer: ```suggestion
The following are notable, non-breaking updates to be aware of:
```

[[breaking-8.13.0]]
[float]
=== Breaking changes

> There are no breaking changes in 8.13.

Reviewer: ```suggestion
There are no breaking changes in 8.13.0.
```
